src/consensus/params.h

```diff
@@ -29,7 +29,12 @@ struct BIP9Deployment {
     int bit;
     /** Start MedianTime for version bits miner confirmation. Can be a date in the past */
     int64_t nStartTime;
-    /** Timeout/expiry MedianTime for the deployment attempt. */
+    /** Timeout/expiry MedianTime for the deployment attempt.
+     * Setting to 0 will prevent advertising presence of softfork,
```

Comment:
Either change to `uint64_t nTimeout` or update description to something like:
```
If greater than 0 then it prevents ...
```
since below you changed to `>` operator and `nTimeout` is not unsigned.